docs/en/developers/learn-appfile.md

```diff
@@ -146,39 +147,33 @@ spec:
 
 ## [Optional] Configure another workload type
 
-By now we have deployed a *webservice* workload. We can also add a *task* workload in appfile:
-
-> Below is a simplified [k8s job example](https://kubernetes.io/docs/concepts/workloads/controllers/job/#running-an-example-job) using Vela:
+By now we have deployed a *[Web Service](references/workload-types/webservice.md)*. We can also add another service of *[Task](references/workload-types/task.md)* type in the same app:
 
 ```yaml
 services:
   pi:
+    type: task
     image: perl 
     cmd: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
```

Comment:
does it really work?

Reply:
I think it's copied from here https://dockerlabs.collabnix.com/kubernetes/cheatsheets/kubectl.html

```
# Start the perl container to compute π to 2000 places and print it out.
kubectl run pi --image=perl --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'
```